src/main/java/PasswordGenerator.java

```diff
@@ -1,8 +1,84 @@
-/*
- * This Java source file was generated by the Gradle 'init' task.
- */
+import com.google.common.annotations.VisibleForTesting;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+import java.util.Scanner;
+
 public class PasswordGenerator {
-    public String generatePassword() {
-        return "a+1";
+
+    public String generatePassword () {
```

Comment:
Feels like this doesn't really generate the password - it reads the user input, so maybe it should be named accordingly?